### PR TITLE
Master - Initial pull request

### PR DESCRIPTION
**Pull request by EricJohnEastwood**

### CODE REVIEW

File: client/app/src/androidTest/java/com/mosip/testapp/ExampleInstrumentedTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
+        // Context of the app under test.
+        Context appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
+        assertEquals("com.mosip.testapp", appContext.getPackageName());
```
Comment (ase-101) [inline client/app/src/androidTest/java/com/mosip/testapp/ExampleInstrumentedTest.java:24]: change the package name to "io.mosip.registration.app"

File: client/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.mosip.testapp">
```
Comment (ase-101) [inline client/app/src/main/AndroidManifest.xml:3]: change the package name to "io.mosip.registration.app"

File: client/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.TestApp">
```
Comment (ase-101) [inline client/app/src/main/AndroidManifest.xml:11]: change TestApp to registration-client

File: client/app/src/androidTest/java/com/mosip/testapp/ExampleInstrumentedTest.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package com.mosip.testapp;
```
Comment (ase-101) [inline client/app/src/androidTest/java/com/mosip/testapp/ExampleInstrumentedTest.java:1]: change the package name to "io.mosip.registration.app"

File: client/app/src/main/java/com/mosip/testapp/MainActivity.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
+package com.mosip.testapp;
```
Comment (ase-101) [inline client/app/src/main/java/com/mosip/testapp/MainActivity.java:1]: change the package name to "io.mosip.registration.app"

File: client/app/src/main/java/com/mosip/testapp/OpenDialog.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
+package com.mosip.testapp;
```
Comment (ase-101) [inline client/app/src/main/java/com/mosip/testapp/OpenDialog.java:1]: change the package name to "io.mosip.registration.app"

File: client/app/src/main/res/values-night/themes.xml [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Base application theme. -->
+    <style name="Theme.TestApp" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
```
Comment (ase-101) [inline client/app/src/main/res/values-night/themes.xml:3]: change "Theme.TestApp" to "Theme.registration-client"

File: client/app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">TestApp</string>
```
Comment (ase-101) [inline client/app/src/main/res/values/strings.xml:2]: Change TestApp to registration-client

File: client/app/src/main/res/values/themes.xml [hunk 9]
```diff
@@ -0,0 +1,16 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Base application theme. -->
+    <style name="Theme.TestApp" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
```
Comment (ase-101) [inline client/app/src/main/res/values/themes.xml:3]: change "Theme.TestApp" to "Theme.registration-client"

File: client/app/src/test/java/com/mosip/testapp/ExampleUnitTest.java [hunk 10]
```diff
@@ -0,0 +1,17 @@
+package com.mosip.testapp;
```
Comment (ase-101) [inline client/app/src/test/java/com/mosip/testapp/ExampleUnitTest.java:1]: change the package name to "io.mosip.registration.app"

**Review by EricJohnEastwood — COMMENTED** [review]
Added the necessary changes. Additionally, "registration-client"  was renamed to "registration_client" following a build failure (naming convention in xml files)

**Pull request merged**